mapper/credentialmapper.go

```diff
@@ -9,15 +9,35 @@ import (
 	"github.com/rancher/norman/types/values"
 )
 
+const credAnn = "io.cattle.cloud.cred_name"
```

Comment:
Do we have any format for adding annotations? I see some annotations with "field.cattle.io/" prefix - should we be following that?

Reply:
I see that using `field.cattle.io` in annotations has its own mapper, mapper/annotation_field.go and I don't think that logic applies here .. I named it similar to `io.cattle.kontainer_driver.name` (which is label but used for internal purposes)